<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Problem Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="View" name="view">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(66, 66, 66);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10,1,40">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>Layer List</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <property name="indent">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="layer_view">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="indentation">
          <number>0</number>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="animated">
          <bool>false</bool>
         </property>
         <attribute name="headerCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="headerDefaultSectionSize">
          <number>80</number>
         </attribute>
         <attribute name="headerMinimumSectionSize">
          <number>80</number>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>Node List</string>
         </property>
         <property name="indent">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="node_view">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="indentation">
          <number>0</number>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <attribute name="headerCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="headerDefaultSectionSize">
          <number>80</number>
         </attribute>
         <attribute name="headerMinimumSectionSize">
          <number>80</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="tool_bar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(3, 169, 244);
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
   <addaction name="separator"/>
   <addaction name="action_preview"/>
   <addaction name="action_screenshot"/>
   <addaction name="separator"/>
   <addaction name="action_select"/>
   <addaction name="action_edge"/>
   <addaction name="separator"/>
   <addaction name="action_delete"/>
   <addaction name="action_undo"/>
   <addaction name="action_redo"/>
  </widget>
  <widget class="QStatusBar" name="status">
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <action name="action_new">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="editor.qrc">
     <normaloff>:/actions/icons/new.png</normaloff>:/actions/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>new</string>
   </property>
   <property name="toolTip">
    <string>new</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="editor.qrc">
     <normaloff>:/actions/icons/save.png</normaloff>:/actions/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="toolTip">
    <string>save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_open">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="editor.qrc">
     <normaloff>:/actions/icons/open.png</normaloff>:/actions/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
   <property name="toolTip">
    <string>open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_preview">
   <property name="icon">
    <iconset resource="editor.qrc">
     <normaloff>:/actions/icons/preview.png</normaloff>:/actions/icons/preview.png</iconset>
   </property>
   <property name="text">
    <string>preview</string>
   </property>
   <property name="toolTip">
    <string>preview</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="editor.qrc">
     <normaloff>:/actions/icons/undo.png</normaloff>:/actions/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>undo</string>
   </property>
   <property name="toolTip">
    <string>undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="editor.qrc">
     <normaloff>:/actions/icons/redo.png</normaloff>:/actions/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>redo</string>
   </property>
   <property name="toolTip">
    <string>redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="editor.qrc">
     <normaloff>:/actions/icons/delete.png</normaloff>:/actions/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
   <property name="toolTip">
    <string>delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_select">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="editor.qrc">
     <normaloff>:/actions/icons/select.png</normaloff>
     <activeon>:/actions/icons/select_checked.png</activeon>:/actions/icons/select.png</iconset>
   </property>
   <property name="text">
    <string>select</string>
   </property>
   <property name="toolTip">
    <string>select</string>
   </property>
   <property name="shortcut">
    <string>Shift+F</string>
   </property>
  </action>
  <action name="action_edge">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="editor.qrc">
     <normaloff>:/actions/icons/edge.png</normaloff>
     <activeon>:/actions/icons/edge_checked.png</activeon>:/actions/icons/edge.png</iconset>
   </property>
   <property name="text">
    <string>edge</string>
   </property>
   <property name="toolTip">
    <string>draw edge</string>
   </property>
   <property name="shortcut">
    <string>Shift+P</string>
   </property>
  </action>
  <action name="action_screenshot">
   <property name="icon">
    <iconset resource="editor.qrc">
     <normaloff>:/actions/icons/screenshot.png</normaloff>:/actions/icons/screenshot.png</iconset>
   </property>
   <property name="text">
    <string>screenshot</string>
   </property>
   <property name="toolTip">
    <string>screenshot</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>View</class>
   <extends>QGraphicsView</extends>
   <header>widgets.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="editor.qrc"/>
 </resources>
 <connections/>
</ui>
